manager/utils_test.go

```diff
@@ -102,10 +103,12 @@ func nothingSignRequest(meta metadata.Metadata, key crypto.PrivateKey, request *
 
 func persistingWriteKeypair(store storage.Interface, clock clock.Clock) WriteKeypairFunc {
 	return func(meta metadata.Metadata, key crypto.PrivateKey, chain []byte, ca []byte) error {
-		store.WriteFiles(meta, map[string][]byte{
+		if err := store.WriteFiles(meta, map[string][]byte{
 			"ca":   ca,
 			"cert": chain,
-		})
+		}); err != nil {
```

Comment:
Huh.... I wonder why we never checked this error before. I think at some point maybe I wrote WriteFiles to never error? 🤔 

+1 from me, though I think we should explicitly call out in the release notes that we now return errors when writing CA and certificate data in case some implementation relied on this behaviour somehow.